packages/@modelfusion-llamacpp-bindings/src/LlamaCppBindingsCompletionModel.ts

```diff
@@ -0,0 +1,253 @@
+import {
+  TextGenerationBaseModel,
+  TextGenerationModel,
+  TextGenerationModelSettings,
+  TextGenerationPromptTemplateProvider,
+  InstructionPrompt,
+  ChatPrompt,
+  BasicTokenizer,
+  FullTokenizer,
+  TextGenerationResult,
+} from "modelfusion";
+import { AbstractModel } from "modelfusion/model-function/AbstractModel";
+
+export interface LlamaCppBindingsCompletionPrompt {
```

Comment:
How similar are prompts and settings to the LlamaCppCompletionPrompt / Settings from modelfusion? Maybe you can reuse those instead? Both works for me, wondering which path is better long term.